source/_integrations/music_assistant.markdown

```diff
@@ -69,6 +69,38 @@ media_content_type: playlist
 media_content_id: 'The Best of Disco'
 ```
 
+### Action `media_player.browse_media`
```

Comment:
This is a media player action, not specific to music assistant.

It thus isn't integration specific and thus shouldn't be documented here but in the media player documentation.

Reply:
But the media player documentation says that the media_content_id is “integration specific” so this section is intended to show the specifics?

Reply:
All text pertaining to HA actions removed. media_content_id info added to the top.